src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/SHA256CryptoServiceProvider.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace System.Security.Cryptography
+{
+    //
+    // If you change this file, make the corresponding changes to all of the SHA*CryptoServiceProvider.cs files.
+    //
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
I think the idea is to discredit CSP, not just insecure algorithms (Aes hasn't been cracked afaik...)